src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -20,8 +20,8 @@ public DateTime StartTime
         {
             get
             {
-                // TODO: Implement this
-                throw NotImplemented.ByDesign;
+                Interop.libproc.rusage_info_v3 info = Interop.libproc.proc_pid_rusage(_processId);
+                return new DateTime(Convert.ToInt64(info.ri_proc_start_abstime));
```

Comment:
For these time-related properties, are these times already in the same units / basis / etc. that DateTime's ctor expects?